### DISCUSSION TITLE
TwitchIO Awards

### DISCUSSION THREAD

**Discussion by QMasterkazna** [TwitchIO/TwitchIO#432] [Questions]
Help, I want to make a function for the event cost of points so that the bot can see who spent the points and what.

**Comment by chillymosh** [comment]
@QMasterkazna You'll need to use pubsub or one of the two eventsub transports. This also only applies to custom reward points and not the built in ones.
There are many examples already on the discord if you care to join.

**Reply by QMasterkazna** [reply]
Yes I would like to look at examples if possible

**Reply by QMasterkazna** [reply]
can you give a link to discord examples?

**Reply by chillymosh** [reply]
The link is in the main TwitchIO github page: https://discord.gg/RAKc3HF
